Examples/Physics_applications/free_electron_laser/analysis_fel.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+"""
+This script tests that the FEL is correctly modelled in the simulation.
+
+The physical parameters are the same as the ones from section 5.1
+of https://arxiv.org/pdf/2009.13645
+
+The simulation uses the boosted-frame technique as described in
+https://www.osti.gov/servlets/purl/940581
+In particular, the effect of space-charge is effectively turned off
+by initializing an electron and positron beam on top of each other,
+each having half the current of the physical beam.
+
+The script checks that the radiation wavelength and gain length
+are the expected ones. The check is performed both in the
+lab-frame diagnostics and boosted-frame diagnostics.
+"""
+
+import os
+import sys
+
+import numpy as np
+from openpmd_viewer import OpenPMDTimeSeries
+from scipy.constants import c, e, m_e
+
+sys.path.insert(1, "../../../../warpx/Regression/Checksum/")
+import checksumAPI
```

Comment:
In #5297 we made all analysis scripts uniform with respect to the checksums parts. The new docs should reflect this (i.e., they should report code snippets that hopefully contributors will copy/paste in the future, keeping the uniformity). For now the uniformity is primarily aimed at making some follow-up changes coming up soon a bit easier to implement across the code base:
```suggestion
from checksumAPI import evaluate_checksum
```